sdk/src/main/scala/com/horizen/SidechainApp.scala

```diff
@@ -63,10 +60,14 @@ class SidechainApp @Inject()
    @Named("WalletForgingBoxesInfoStorage") val walletForgingBoxesInfoStorage: Storage,
    @Named("ConsensusStorage") val consensusStorage: Storage,
    @Named("CustomApiGroups") val customApiGroups: JList[ApplicationApiGroup],
-   @Named("RejectedApiPaths") val rejectedApiPaths : JList[Pair[String, String]]
+   @Named("RejectedApiPaths") val rejectedApiPaths : JList[Pair[String, String]],
+   val sidechainCoreTransactionFactory : SidechainCoreTransactionFactory,
+   val sidechainTransactionsCompanion : SidechainTransactionsCompanion
```

Comment:
SimpleApp is broken. Please, run  python tests full suit and fix the issues.

Reply:
fixed